samples/Antiforgery.Angular1/Antiforgery.Angular1.csproj

```diff
@@ -19,4 +19,8 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="$(AspNetCoreVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
```

Comment:
Should have a general samples.props file or include common.props in all samples; this is boiler plate. We do that in other repos too.